src/puppetlabs/dujour/version_check.clj

```diff
@@ -109,6 +110,14 @@
       (log/info update-msg))
     response))
 
+(defn get-hash
```

Comment:
Did a quick check just to make sure this is properly sha-512'ing strings. Looks to be the case!

```shell
brian@localhorse:dujour-version-check % echo -n "localhorse.local" | shasum -a 512                                                           ±[CERTNAME-HASH]
a89134338a77c56dc0c475c7dd1bf41c8647ea515b045ef40d644db89ede774d332d443f98af08b2634b6db49a3a35a57e4a0b59bf6f8df75709787d84fd43f2  -
```

```clojure
user=> (use 'puppetlabs.dujour.version-check :reload)
nil
user=> (def my-hostname-hash "a89134338a77c56dc0c475c7dd1bf41c8647ea515b045ef40d644db89ede774d332d443f98af08b2634b6db49a3a35a57e4a0b59bf6f8df75709787d84fd43f2"
  #_=> )
#'user/my-hostname-hash
user=> (= my-hostname-hash (puppetlabs.dujour.version-check/get-hash "localhorse.local"))
true
```